### PR TITLE
require the json

### PR DESCRIPTION
**Pull request by kentcdodds**
I tried to follow your coding standards as closely as possible :-)

horse-applause-clapping-8RxCFgu88jUbe

### CODE REVIEW

**Comment by kentcdodds** [comment]
Just realized this is a breaking change. Before you were logging the JSON string, now it's logging an object. Would you like me to update the PR to JSON.stringify it? ;-)

**Comment by kentcdodds** [comment]
Also, I should apologize. This PR is about as "Well, actually..." as you can get

**Comment by elijahmanor** [comment]
Yeah, `JSON.stringfy` with some formatting options would be nice... like

`console.log( JSON.stringify( require( './me.json' ), null, 3 ) );`

Thanks for the PR btw. Hadn't thought of a `require`, like that much better

**Comment by elijahmanor** [comment]
Also, I guess I should consider adding a `CONTRIBUTING.md` file for future PRs ;)

**Comment by kentcdodds** [comment]
Updated :+1:

**Comment by elijahmanor** [comment]
@kentcdodds strange, not seeing your updates. did you push your changes? i'm fine w/ a squash if you want to do that

**Comment by kentcdodds** [comment]
Must have forgotten to stage my changes.

page-crimson-shaft-qvVJ36yWjbq24

**Comment by kentcdodds** [comment]
Boom, there you go :-)

File: index.js [hunk 1]
```diff
@@ -1,9 +1 @@
-var fs = require( "fs" );
-
-fs.readFile( "./me.json", "utf8", function( err, data ) {
-  if ( err ) {
-    return console.log( err );
-  }
-  console.log( data );
-} );
-
+console.log( JSON.stringify( require( './me.json' ), null, 3 ) );
```
Comment (sarbbottam) [inline index.js:1]: indenting with `3 spaces`? Is that a typo or on purpose?

Comment (kentcdodds) [inline index.js:1]: It's what the doctor ordered. I wasn't going to say anything but.... (－‸ლ)

**Comment by elijahmanor** [comment]
ha ha @sarbbottam @kentcdodds :) thanks for the update, i'm about to merge! gracias

**Pull request merged**